Resources/Prototypes/Recipes/Crafting/Graphs/improvised/improvised_shotgun.yml

```
@@ -11,11 +11,24 @@
           sprite: Structures/Piping/Atmospherics/pipe.rsi
           state: pipeStraight
         name: pipe
-      - tag: ModularReceiver
+        # Moffstation - Begin - Remove modular receiver requirement for improv shotgun, make it easier to build
+#      - tag: ModularReceiver
+#        icon:
+#          sprite: Objects/Misc/modular_receiver.rsi
+#          state: icon
+#        name: modular receiver
+      - tag: Screwdriver
         icon:
-          sprite: Objects/Misc/modular_receiver.rsi
-          state: icon
-        name: modular receiver
+          sprite: Objects/Tools/screwdriver.rsi
+          state: screwdriver
+        name: screwdriver
+      - material: Steel
+        amount: 1
+        icon:
+          sprite: Objects/Materials/Sheets/metal.rsi
+          state: steel
+        name: steel
```

```suggestion
        name: screwdriver
      - material: Steel
        amount: 1
```
This seems to be the test failure.
Something about since tags are really nebulous, the construction menu wants an icon and name -- but a material is always tied to an entity stack thingy? Unsure, but these're the offending lines.